app/js/preferences/partialValueFunctionController.js

```diff
@@ -115,8 +115,8 @@ define(['angular', 'lodash', '../controllers/wizard'], function(angular, _, Wiza
             to: to
           },
           sliderOptions: {
-            floor: Math.min(from, to) + 0.001,
-            ceil: Math.max(from, to) - 0.001,
+            floor: Math.min(from, to) + Math.abs(to - from) / 100,
```

Comment:
Does this need significant digit messing about?

Reply:
The number filter is applied for displayed values in html.